deployment/terraform/iam.tf

```diff
@@ -121,6 +121,12 @@ resource "aws_iam_role_policy_attachment" "ec2_service_role" {
   policy_arn = "${var.ec2_service_role_policy_arn}"
 }
 
+resource "aws_iam_role_policy" "ses_send_email_from_batch" {
+  name       = "ses${var.environment}EmailSendingPolicy"
+  role       = "${aws_iam_role.container_instance_ec2.name}"
+  policy     = "${data.aws_iam_policy_document.ses_send_email.json}"
+}
```

Comment:
By using the [`aws_iam_role_policy`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy) resource, we're creating an inline policy attached to the container instance role. It reuses the contents of the `ses_send_email` policy document, but it is creating separate inline policy objects for our application container role and our Batch container role!